environments/softlayer/app-processes.yml

```diff
@@ -18,28 +18,26 @@ celery_processes:
     ucr_indicator_queue:
       concurrency: 1
     celery,export_download_queue:
-      concurrency: 4
+      concurrency: 2
       max_tasks_per_child: 5
     saved_exports_queue:
       concurrency: 2
       max_tasks_per_child: 1
       optimize: True
     reminder_case_update_queue:
       pooling: gevent
-      concurrency: 5
-      num_workers: 2
+      concurrency: 1
```

Comment:
hey this changes 2 workers (each with concurrency of 5, but via gevent not forked processes) to one worker with concurrency 1 (effective concurrency of 10 -> 1). same thing with the `reminder_queue` that seems a pretty dramatic reduction. I also do not believe there is a large memory overhead in gevent processes (although could be wrong). Might be better start by moving it to one worker with concurrency 5 instead of both slashing the number of workers and their concurrency

Reply:
Cool, Updated.